src/utils/input-utils.ts

```diff
@@ -317,9 +317,11 @@ export const VALIDATE_VALUE_BY_TYPE: {
 } = {
   'int': integerFieldValidation,
   'integer2': integer2FieldValidation,
+  'int4': integer4FieldValidation,
   'integer4': integer4FieldValidation,
   'integer8': integer8FieldValidation,
   'float': numericFieldValidation,
+  'float4': numericFieldValidation,
```

Comment:
same comment about these types if we use the type object instead with getInputType